dc-cudami-editor/src/state/Reducer.ts

```diff
@@ -34,7 +83,8 @@ const Reducer = (state: State, action: Action): State => {
         ...state,
         dialogsOpen: {
           ...state.dialogsOpen,
-          [action.payload.name]: !state.dialogsOpen[action.payload.name],
+          [action.payload.type]:
+            !state.dialogsOpen[action.payload.type as DialogName],
```

Comment:
```suggestion
            !state.dialogsOpen[action.payload.type],
```
falls es oben mit `PayloadTypes` beim action-Parameter funktioniert